index.js

```diff
@@ -107,26 +107,28 @@ async function authorize(options) {
   return fetch(`${ims}/ims/exchange/jwt/`, postOptions)
     .catch(e => throwRequestFailedError(e.message))
     .then(res => {
-      if (res.ok) {
-        return res.json();
-      } else {
-        throwRequestFailedError(res.statusText);
-      }
+      return res.json().then(data => {
```

Comment:
I think it's desirable to let `res.json` throw if there's something wrong, This gives us a stack trace and insight that there was something wrong doing the exchange. In this scenario, we would already have made the roundtrip, so let's figure out why it blew up.